pinot-controller/src/main/java/org/apache/pinot/controller/api/access/BasicAuthAccessControlFactory.java

```diff
@@ -50,10 +51,16 @@
 
   private AccessControl _accessControl;
 
+  @Override
   public void init(PinotConfiguration configuration) {
     _accessControl = new BasicAuthAccessControl(BasicAuthUtils.extractBasicAuthPrincipals(configuration, PREFIX));
   }
 
+  @Override
+  public void init(PinotConfiguration pinotConfiguration, PinotHelixResourceManager pinotHelixResourceManager) {
+    init(pinotConfiguration);
+  }
+
```

Comment:
same as above. no changes should happen to this file

Reply:
Sure. I have canceled all changes to this file.

Reply:
Sure. I have canceled all changes to this file.